job-service/src/test/java/com/hpe/caf/services/job/jobtype/DefaultDefinitionParserTest.java

```diff
@@ -15,25 +15,26 @@
  */
 package com.hpe.caf.services.job.jobtype;
 
-import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.databind.node.IntNode;
-import com.fasterxml.jackson.databind.node.NullNode;
-import com.fasterxml.jackson.databind.node.TextNode;
-import com.hpe.caf.services.configuration.AppConfig;
-import com.hpe.caf.services.job.api.generated.model.WorkerAction;
-import com.hpe.caf.services.job.exceptions.BadRequestException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 import org.mockito.Mockito;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.node.IntNode;
+import com.fasterxml.jackson.databind.node.NullNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.hpe.caf.services.configuration.AppConfig;
+import com.hpe.caf.services.job.api.generated.model.WorkerAction;
+import com.hpe.caf.services.job.exceptions.BadRequestException;
```

Comment:
Remove the unnecessary re-ordering of the imports.

Reply:
Done